src/scales/quantitativeScale.ts

```diff
@@ -42,6 +51,44 @@ module Plottable {
       return super.domain(values); // need to override type sig to enable method chaining:/
     }
 
+    /**
+     * Gets the minimum domain value when autoDomain()-ing. 
+     */
+    public min(): D;
+    /**
+     * Sets the minimum domain value when autoDomain()-ing.
+     * 
+     * @returns {QuantitativeScale<D>} The calling QuantitativeScale.
+     */
+    public min(value: D): QuantitativeScale<D>;
+    public min(value?: D): any {
+      if (value === undefined) {
```

Comment:
In this case, passing `null` is used to remove a `min()`. I'll add it to the docs so it's clear.